rough outline for aspects intro (#87)
MiniFate/MiniFate: _chapters/00-outline.md

## Who You Need

To play {GameName}, you need one GM (game master) and at least two players.
Each player takes on the role of a PC (player character).
The GM is responsible for the world around the PCs, including its cast of NPCs (non-player characters).

The GM and PCs are not opponents, even though the PCs and NPCs will often be enemies.
Play is collaborative.
Everyone at the table has the same goal: telling a good story and having a good time.

## What You Need

> <!-- todo: don't start on a negative. and can we do this in two paragraphs? -->

Reviewer: ```suggestion
```

> {GameName} doesn't require a lot to get started:

Reviewer: ```suggestion
{GameName} requires a few things to get started:
```

> just pens to write with,

Reviewer: ```suggestion
pens to write with,
```

Before your first session, the group decides they want their game to be "steampunk Robin Hood".
As it turns out, that can mean a bunch of different things. <!-- lose this line? -->
To {A}, that means a swashbuckling adventure where bandits thumb their noses at the lecherous sheriff.

> {B} is picturing something gritty; soldiers return from a holy war to find that England no longer feels like home. <!-- is dystopian steampunk allowed? or is steampunk supposed to be optimistic? -->

Reviewer: Sure!

Although honestly I'm not really getting into the Robin Hood grove. I'll see how today's aspect writing goes. :-/

Author: I'll avoid investing further in it for the moment. Focus on non-setting text, and maybe feel out going back to TBY and co

{C} doesn't much care about the setting, as long as she can play as a wizard.
And {D}, thinking back to the childhood movie, is looking forward to the talking animals.

<!-- passive voice or not? probably go through and make this consistent at the end -->
Before starting the game, everyone needs to be on the same page.
{GameName} provides a powerful tool for that: aspects.

<!--
Alliterative player and PC names?
Easy to keep track of so we can introduce them once, use player and PC names as convenient.

Rowan plays Robin Hood
Morgan plays Maid Marian
Lee/Jordan plays Little John
Frankie/Taylor plays Friar Tuck
Gabby is the GM
-->

## Core Concept: Aspects

At face value, an aspect is just a short phrase written down for the whole table to see.

- aspects are short improvised phrases. can be about anything: people, places, things, relationships, scenes, overall setting.
- aspects are true. otherwise, rewrite it or get rid of it.

> - aspects keep everyone on the same page about what's important. the focus of the scene. like lighting or music or director's notes in a movie. depending on the framing, FOOBAR can be tense or sad or sexy.

Reviewer: ```suggestion
- aspects keep everyone on the same page about what's important. the focus of the scene. like lighting or music or director's notes in a movie. depending on the framing, TODO FOOBAR can be tense or sad or sexy.
```

- aspects provide scaffolding for building out the story/world. introduce things that you can come back to later: people, places, relationships, technologies. `Hunted by the Sheriff of Nottingham`, `Disheartened Veteran of the Tenth Crusade` `something flying boat something`
- aspects provide bonuses on rolls. there's a chapter about that later. probably do NOT introduce fate points here.

> - writing good aspects. you can lean on inside jokes and pop culture.

Reviewer: Love that we're going to have this section.
